.github/workflows/docker.yml

```diff
@@ -0,0 +1,31 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
this action will be triggered on every of push to main ?

Reply:
You know it shouldn't be 🙂 thank you for your review I will be making the changes now